satpy/scene.py

```diff
@@ -37,6 +38,12 @@
 from satpy.resample import get_area_def, prepare_resampler, resample_dataset
 from satpy.writers import load_writer
 
+try:
+    import hvplot.xarray  # noqa
+except ImportError:
+    hvplot.xarray = None
```

Comment:
This will raise `NameError` if `hvplot` isn't installed. I think this needs to be:

```suggestion
try:
    import hvplot.xarray  as hvplot_xarray  # noqa
except ImportError:
    hvplot_xarray = None
```

And in `.to_hvplot use `if hvplot_xarray is None` where availability is checked and `ImportError` is raised.